### PR TITLE
Get and Post author and message

### PR DESCRIPTION
**Pull request by johan-t**
## Describe the issue:
To add the description field we need to be able to post and get the messages.

## Explain how you solved the issue:
I added the parameters to the structs and to the sql requests.

## Additional notes or considerations:
- I took a long time to try to refactor the two ticketInfo structs into one until I have realized that it is not possible. This 3 hour detour could have been avoided if we had written better code and documented our decisions better. I think there are a couple of things we should try to do:

  1. **Better comments**: Instead of explaining what something is, explain why we did it this way. This is especially important for unorthodox decisions. Also we should write comments so that when we highlight the element we see it in the documentation field of vs code
  2. **Better Naming**: From the name of the structs it was not clear that they are two structs with a different purpose, as they were both synonyms of each other but had a very different purpose. 
  3. **Consistency**: If we have standard practices it is easier to identify if something is sloppy or if there is a purpose behind something.
  
I tried to improve the code. Take a look and feel free to point out something where we could improve on to avoid the things that have happened to me.

- I have also added the author id as we need this later and we will have an easier time later if it is already implemented.

### CODE REVIEW

**Comment by johan-t** [comment]
@johan-t remember to change this so that we are only sending the message if author is none to avoid sending conflicting info when the telegram bot did a misclassification

File: database/database.go [hunk 1]
```diff
@@ -236,11 +241,12 @@ func GetHistoricStations(timestamp time.Time, remaining int, maxRecursiveCalls i
...
-            FROM ticket_info
-            WHERE timestamp >= NOW() - INTERVAL '15 minutes'
-            AND station_name IS NOT NULL
+func GetLatestTicketInspectors() ([]utils.TicketInspector, error) {
+	sql := `SELECT timestamp, station_id, direction_id, line,
+			CASE WHEN author IS NULL THEN message ELSE NULL END as message
+			FROM ticket_info
+			WHERE timestamp >= NOW() - INTERVAL '15 minutes'
+			AND station_name IS NOT NULL
 			AND station_id IS NOT NULL;`
```
Comment (brandesdavid) [inline database/database.go:250]: good job. if the frontend receives null station_id, it breaks

File: api/postInspector.go [hunk 2]
```diff
@@ -63,21 +64,31 @@ func processRequestData(req structs.InspectorRequest) (*structs.ResponseData, er
...
+		messagePtr = &req.Message
+		response.Message = req.Message
+	}
+
 	now := time.Now()
 
-	log.Printf("Inserted ticket info: %v", data)
+	log.Printf("Inserted ticket info: %v", response)
 
 	// Directly pass the pointers for all parameters.
```
Comment (brandesdavid) [inline api/postInspector.go:87]: remove comment

**Review by brandesdavid — APPROVED** [review]
lgtm

**Pull request merged**